sirepo/util.py

```diff
@@ -218,6 +217,10 @@ def random_base62(length=32):
     return ''.join(r.choice(numconv.BASE62) for x in range(length))
 
 
+def requests_not_found(request):
```

Comment:
I did requests because it is a requests request (a request made by the requests package). I did this to differentiate from other requests we might have (ex tornado, flask). Thoughts?

Reply:
Ok, I tried encapsulating in `_request`. e84986d0